TTHAnalysis/python/analyzers/isoTrackDeDxAnalyzer.py

```diff
@@ -88,9 +91,16 @@ def process(self, event):
                 for ih in xrange(nhits):
                     pxclust = dedx.pixelCluster(ih)
                     if not pxclust: continue
-                    dedxArray[ih] = pxclust.charge()
+                    
+                    dedxArray[ih] = pxclust.charge()/dedx.pathlength(ih)
+                    
+                    # convert number of electrons to MeV
+                    if ih < 5: dedxArray[ih] *= pixelChargeToEnergyCoefficient
+                    else:      dedxArray[ih] *= stripChargeToEnergyCoefficient
```

Comment:
we're in this loop only if there's a pixel cluster (see above line 93), so it's always pixelChargeToEnergyCoefficient.
for strips, if we want them, we need to get the strip cluster instead